payu/archive_scripts/archive_cice_restarts.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash -i
+# clean up cice_restarts.sh
+
+latest_o=$(ls -drv archive/output*[0-9] | head -1)
+
+if [ -f $latest_o/access-om3.cice.r.* ]
```

Comment:
Do we want to update this to allow for multiple restarts ?

Reply:
When would that be needed? Are we likely to need it?

Reply:
Currently, if configured to save extra restarts (i.e. those without "restarting" the model), this line will just fail, and not remove the restart file which was used for to start the current model run.

Reply:
If it's easy to change to handle that case robustly, then I guess there's no reason not to